install.sh

```diff
@@ -164,6 +165,18 @@ create_tfstate_resources() {
   then
     log "S3 bucket '${S3_BUCKET}' already exists."
   else
+    # Check if the logging bucket is existed otherwise stop the installation
+    LOGGING_BUCKET=$(get_variable 'logging_bucket' "${CONFIG_ABS_PATH}")
+    if [ -n "${LOGGING_BUCKET}" ]; then
+      set +e
+      aws s3api head-bucket --bucket "${LOGGING_BUCKET}" 2>/dev/null
+      LOGGING_BUCKET_EXISTS=$?
```

Comment:
I think `head-bucket` command is what we need here. It may `get-bucket-location` can mimic what the `head-bucket` does but it will be hacky solution while we have a dedicated command to determine if a bucket is existed and the account has permission to use it or not. 

Here is the definition of the `head-bucket` which is exactly what we try to do here:

> This action is useful to determine if a bucket exists and you have permission to access it. The action returns a 200 OK if the bucket exists and you have permission to access it.

https://docs.aws.amazon.com/cli/latest/reference/s3api/head-bucket.html

Reply:
Yup that sounds like a better way of checking it.

Reply:
Perfect